src/runtime.js

```diff
@@ -45,14 +45,6 @@ function getNativeTypeSize(type) {
 
 var Runtime = {
   getNativeTypeSize: getNativeTypeSize,
-
-  //! Returns the size of a structure field, as C/C++ would have it (in 32-bit,
-  //! for now).
-  //! @param type The type, by name.
-  getNativeFieldSize: function(type) {
-    return Math.max(getNativeTypeSize(type), Runtime.QUANTUM_SIZE);
-  },
-
```

Comment:
Again I'm not sure how to remove this function safely since its conceivable that somebody out there is using this in a library.js file.

Reply:
I think it's fine to remove this but please mention in the changelog, with an explanation of the replacement (call `Runtime.getNativeTypeSize` instead and max up to 4).

Reply:
I decided to play it safe and not actually remove the function.. and instead try to come up with a consistent way to deprecate/remove stuff.    See: https://github.com/emscripten-core/emscripten/issues/15242